5-network/01-fetch/01-fetch-users/solution.md

```diff
@@ -1,11 +1,11 @@
 
-To fetch a user we need: `fetch('https://api.github.com/users/USERNAME')`.
+Aby pobrać użytkownika wykorzystamy: `fetch('https://api.github.com/users/USERNAME')`.
```

Comment:
```suggestion
Do pobrania użytkownika wykorzystamy: `fetch('https://api.github.com/users/USERNAME')`.
```
Konstrukcje z "Aby..." brzmią słabo w krótkich zdaniach, a dobrze w nieco bardziej złożonych. Lepiej to uprościć.